src/pages/login.vue

```diff
@@ -33,13 +53,13 @@ export default {
     }
   },
   computed: {
-    ...mapGetters(['configuration'])
+    ...mapGetters(['configuration']),
+    backgroundStyle () {
+      return `background-image: linear-gradient(${this.configuration.theme.colors.primary}, ${this.configuration.theme.colors.secondary});`
```

Comment:
Search Engine's don't like this inline styling, this can result in bad seo performance for your OC login page :nerd_face: 
Just kidding, I like that styling :+1: